.github/workflows/build.yml

```diff
@@ -61,47 +68,102 @@ jobs:
         uses: urkle/action-get-ninja@v1
 
       - name: Configure CMake
+        if: matrix.app_image != true
+        run: |
+          cmake -S . -B ${{ env.BUILD_DIR }} -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -DCMAKE_BUILD_TYPE=Debug -G Ninja
+
+      - name: Configure CMake for AppImage
+        if: matrix.app_image == true
         run: |
-          cmake -S . -B build -DCMAKE_INSTALL_PREFIX=install -DCMAKE_BUILD_TYPE=Debug -G Ninja
+          cmake -S . -B ${{ env.BUILD_DIR }} -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Debug -DLauncher_LAYOUT=lin-system -G Ninja
 
       - name: Build
         run: |
-          cmake --build build
+          cmake --build ${{ env.BUILD_DIR }}
 
       - name: Install
+        if: matrix.app_image != true
+        run: |
+          cmake --install ${{ env.BUILD_DIR }}
+
+      - name: Install for AppImage
+        if: matrix.app_image == true
+        run: |
+          DESTDIR=${{ env.INSTALL_DIR }} cmake --install ${{ env.BUILD_DIR }}
+
+      - name: Download linuxdeploy family
```

Comment:
I think this should be moved up. We don't want to download dependencies after we compiled already. If this task fails we don't want to waste time building

Reply:
Done!